astro/src/content/docs/lifecycle/authenticate-users/oauth/endpoints.mdx

```diff
@@ -1197,39 +1130,6 @@ Ensure you are sending these parameters as form encoded data in the request body
   </APIField>
 </APIBlock>
 
-#### Request Headers
```

Comment:
Was this section duplicated?

Reply:
Yes. It duplicated the request headers and parameters from the `Validate and retrieve a user_code using client credentials` example after providing the correct headers and parameters for using an API key.